### ISSUE TITLE
[Enhancement] fill/stroke color for each point on a Two.Points

### ISSUE THREAD

**Issue by joex92** [jonobr1/two.js#553]
Hi,
I like a lot the `Two.Points` object, but it works practically the same as `Two.Path` with "unconnected" vertices, at least in the final product.
So I believe that on a `Two.Points` object, each vertices could have its own fill and stroke color set... I mean, at least that's what I'm looking for to be able to do... 
This way a `Two.Point` object would be needed, which could be a `Two.Vector` with color properties... ish? idk...

**Comment by jonobr1** [comment]
Yeah, this is a good point and we should add this in the future. At this point in time though you'll have to make different `Two.Points` objects for each set of colors you want to use.
